pandas/tests/frame/test_arithmetic.py

```diff
@@ -1732,6 +1732,18 @@ def test_align_int_fill_bug(self):
         expected = df2 - df2.mean()
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "op", [DataFrame.add, DataFrame.sub, DataFrame.mul, DataFrame.div]
```

Comment:
Cover your changes any additional functions?

Reply:
I went through and double checked the functions, these are the four that are impacted.